access-control-app/docs/api.md

```diff
@@ -873,6 +875,41 @@ Content-Type: application/json;charset=ISO-8859-1
 ** `user_id` - The user whose permissions will be computed.
 ** `user_type` - Either "guest" or "registered".
 
+### <a name="get-s3-buckets"></a> User Access to S3 Buckets
+
+This endpoint will return a list of S3 buckets a user has access to. If a list of providers is included in the request, the list of S3 buckets will be filtered to only include S3 buckets contained by collections within those providers.
+
+#### Parameters
+
+* Required:
+** `user_id` - The user whose available S3 buckets and object prefix names will be determined for.
+
+* Optional
+** `provider[]` - A list of provider ids to filter the S3 bucket and object prefix names by. If not specified, all providers will be included.
+
+
+Example request:
+```
+curl -i -H "Echo-Token: XXXX" "%CMR-ENDPOINT%/s3-buckets?user_id=user1
+
+HTTP/1.1 200 OK
+Content-Length: 81
```

Comment:
Include Content-Type here to make it clear that the result format is application/json.